src/internal/connector/onedrive/restore.go

```diff
@@ -262,13 +262,18 @@ func RestoreCollection(
 						continue
 					}
 
-					deets.Add(
+					err = deets.Add(
 						itemPath.String(),
 						itemPath.ShortRef(),
 						"",
 						"", // TODO: implement locationRef
 						true,
 						itemInfo)
+					if err != nil {
+						// Not critical enough to need to stop restore operation.
+						logger.Ctx(ctx).Infow("accounting for restored item", "error", err)
```

Comment:
I know this code was already reviewed once, but just had a comment. Do you think it might be better to make it a warning instead of an info as this is not something that should be ideally happening.